modules/porous_flow/doc/content/source/bcs/PorousFlowSinkBC.md

```diff
@@ -0,0 +1,14 @@
+# PorousFlowSinkBC
+
+Proxy class that allows users to use the BC syntax for specifying an enthalphy sink.
+This class adds a (PorousFlowSink)[PorousFlowSink.md] and a (PorousFlowEnthalpySink)[PorousFlowEnthalpySink.md].
+
+Users are required to specify:
+
+- `boundary` - The list of boundary IDs from the mesh where this boundary condition applies
+- `pressure` - Pressure
```

Comment:
Important: this should not be "Pressure" but should be "Name of the porepressure variable".   And, I also think it should be rename `porepressure_var` or something similar.

Reply:
✅